Scripts/Swift/iOSTemplateMaker/Sources/iOSTemplateMaker/SetUpiOSProject.swift

```diff
@@ -123,7 +123,7 @@ class SetUpIOSProject {
         if minimumVersion.isEmpty {
             tryMoveDown()
 
-            let defaultVersion = "14.0"
+            let defaultVersion = "15.0"
```

Comment:
We can go with 16.0; only 2% of the share market for the lower version than 16.0

https://developer.apple.com/support/app-store/